taichi_glsl/scalar.py

```diff
@@ -237,3 +238,18 @@ def isinf(x):
         The return value is computed as `2 * x == x and x != 0`.
     '''
     return 2 * x == x and x != 0
+
+
+@ti.pyfunc
+def Truth(cond):
+    '''
+        return 1 if condition is not false, return 0 vice versa
+    :param x:
+
+    :return:
+        The return value is computed as `return abs(cond != 0)`
+        note: Currently Taichi use -1 to represent True.
+        This function serves as a helper function to those who want 1 to represent True
+
+    '''
+    return abs(cond != 0)
```

Comment:
How about:
1 if cond else 0
so that even if taichi use 0xffff for true this works.